.github/workflows/ci.yml

```diff
@@ -103,8 +103,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
-        nautobot-version: ["1.0.3", "1.1.4"]
+        python-version: ["3.7", "3.8", "3.9"]
+        nautobot-version: ["1.2.11", "1.3.8"]
```

Comment:
```suggestion
        nautobot-version: ["1.2.11", "stable"]
```

Reply:
@chadell the error is back on the stable front. Is there something else we want to cover on this before just setting it back?